core/object/script_language.h

```diff
@@ -321,10 +322,10 @@ class ScriptLanguage : public Object {
 		CODE_COMPLETION_KIND_MAX
 	};
 
+	/* /scene/gui/code_edit.h - CodeEdit::CodeCompletionLocation */
 	enum CodeCompletionLocation {
-		LOCATION_LOCAL = 0,
-		LOCATION_PARENT_MASK = 1 << 8,
-		LOCATION_OTHER_USER_CODE = 1 << 9,
+		LOCATION_LOCAL = 1 << 0,
+		LOCATION_BASE = 1 << 1,
 		LOCATION_OTHER = 1 << 10,
 	};
```

Comment:
Changing this enum is technically a compatibility breakage. Extensions using the current version won't work with this change.

Reply:
back tracked this change so that itwouldn't be changed on `CodeEdit` side either.